### PR TITLE
Add integration with Travis CI

### LINKED ISSUE #7: Add integration with Travis CI
Please configure integration with TravisCI:
* [x] [Enable repo testing in TravisCI](https://travis-ci.com/kpi-web-guild/django-girls-blog-RootenberG)
* [x] Add TravisCI badge (master branch) to `README.md` with a single commit. (click Build Status next to octocat icon on [this page](https://travis-ci.com/kpi-web-guild/django-girls-blog-RootenberG))
* [x] Create new branch called `feature/travis-ci-integration` and do all the following steps within that branch.
* [x] Add `.travis.yml` using `pre-commit` as a script. Set it up to run Python 3.7 env and add PIP caching. Ensure you install `pre-commit` before running tests:
```yaml
language: python
python: 3.7
cache: pip
before_install: pip install -U pre-commit
script: pre-commit run --all-files
```
* [x] Push the branch and create a Pull Request for it.

### PR DESCRIPTION
**Pull request by RootenberG**
Closes #7

### CODE REVIEW

**Comment by webknjaz** [comment]
@RootenberG you need to use `Closes #xxx` to refer to the relevant issue

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+matrix:
+    include:
```
Comment (webknjaz) [inline .travis.yml]: Plz use double space indentation for mappings
```suggestion
  include:
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        - python: 3.6
```
Comment (webknjaz) [inline .travis.yml]: And zero indentation for sequence items
```suggestion
    - python: 3.6
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+          dist: trusty
+          sudo: false
+        - python: 3.7
+          dist: xenial
+          sudo: true
```
Comment (webknjaz) [inline .travis.yml]: `sudo` option is long deprecated. It used to control whether Travis will spawn full VMs or docker containers for jobs. But now it only supports VMs.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+language: python
+matrix:
```
Comment (webknjaz) [inline .travis.yml]: You don't really need a matrix. The task was to have just Python 3.7 there.
But if you want both, there's matrix expansion for certain options like:
```suggestion
python:
- "3.7"
- "3.6"
```

**Comment by RootenberG** [comment]
> @RootenberG you need to use `Closes #xxx` to refer to the relevant issue

Of course, my bad.

**Review by webknjaz — CHANGES_REQUESTED** [review]
The first commit looks ok

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    include:
+        - python: 3.6
+          dist: trusty
+          sudo: false
+        - python: 3.7
+          dist: xenial
+          sudo: true
+cache: pip
+before_install: pip install -U pre-commit
+script: pre-commit run --all-files
```
Comment (DrEdi) [inline .travis.yml]: No new line at the end of file :)

**Review by DrEdi — CHANGES_REQUESTED** [review]

**Review by webknjaz — DISMISSED** [review]
I prefer history to be cleaner. But other than that it's ok.

**Comment by webknjaz** [comment]
Rebase this after merging #15

**Comment by webknjaz** [comment]
Rebase this

**Comment by RootenberG** [comment]
> Rebase this

Done

**Comment by webknjaz** [comment]
> Done

Not done. (https://travis-ci.com/kpi-web-guild/django-girls-blog-RootenberG/builds/119212844)

**Review by webknjaz — APPROVED** [review]

**Pull request merged**